torch/quantization/fuse_modules.py

```diff
@@ -48,6 +48,7 @@ def fuse_conv_bn_relu(conv, bn, relu):
     assert(conv.training == bn.training == relu.training),\
         "Conv and BN both must be in the same mode (train or eval)."
     is_3d = isinstance(conv, torch.nn.Conv3d)
+    is_1d = isinstance(conv, torch.nn.Conv1d)
```

Comment:
For the training part, you can do this:
map_to_fused_module_train = {
torch.nn.Conv2d: torch_fused.ConvReLU2d,
torch.nn.Conv3d: torch_fused.ConvReLU3d,
}
fused_module = map_to_fused_module_train.get(type(conv))
if fused_module is not None:
      return fused_module(conv,bn,relu)
else:
      raise NotImplementedError("Cannot fuse modules: {}".format((conv, bn, relu)))